packages/remix/src/generators/library/schema.json

```diff
@@ -44,6 +44,16 @@
       "description": "Should the library be buildable?",
       "default": false
     },
+    "unitTestRunner": {
```

Comment:
question: if the code defaults to Jest, should we also set the default option to jest in the schema?

Reply:
So, all the Stacks that Remix offer right now all use vitest, so i think it might actually be better to default to that if we add a default, what do you think?

Reply:
Looks pretty similar to what we're generating
https://github.com/epicweb-dev/testing-web-apps/blob/main/exercises/04.unit/01.solution.unit/vitest.config.ts

Reply:
We may need this: https://github.com/epicweb-dev/testing-web-apps/blob/main/exercises/04.unit/01.solution.unit/other/test-setup/setup-test-env.ts

or at least the `installGlobals` part of it